OrbitCore/LinuxUtils.cpp

```diff
@@ -86,39 +88,52 @@ std::string ExecuteCommand(const char* a_Cmd) {
 //-----------------------------------------------------------------------------
 void ListModules(pid_t a_PID,
                  std::map<uint64_t, std::shared_ptr<Module>>& o_ModuleMap) {
```

Comment:
should be pointer to map... drop "o_" while you are here.

Reply:
done